canvas_modules/harness/test_resources/parameterDefs/checkboxset_paramDef.json

```diff
@@ -571,5 +571,28 @@
       }
     }
   ],
-  "resources": {}
+  "resources": {
```

Comment:
We don't want descriptions for all values.  I would probably add it to 1 or 2.

Reply:
Left only 2 descs along with the radio resource.